docs/installingLocal.md

```diff
@@ -33,6 +33,8 @@ By default, PrairieLearn does not monitor for server-side changes, so it will **
 docker run -it --rm -p 3000:3000 -e NODEMON=true -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn
 ```
 
+**NOTE**: On MacOS with "Apple Silicon" (ARM64) hardware, the use of R is not currently supported.
```

Comment:
Maybe shift this to just after: https://github.com/PrairieLearn/PrairieLearn/blob/c0c0e638dda4e3f0cc8a921bdc9fd6ef6628279b/docs/installing.md#L42 ?